src/mruby-zest/example/ZynHeaderParams.qml

```diff
@@ -13,7 +13,7 @@ Widget {
         }
     }
     HSlider {type: :float; extern: "/Volume"; height: 0.8; label: "volume"; }
-    Button  {layoutOpts: [:no_constraint]; label: "nrpn" }
```

Comment:
@friedolino78 Did you accidentally remove this line when rebasing? Or is the removal really wanted for this PR?

Reply:
> @friedolino78 Did you accidentally remove this line when rebasing? Or is the removal really wanted for this PR?

I think that button had no function, and i needed some space ;-)